app/controllers/admin/api/services_controller.rb

```diff
@@ -61,7 +61,7 @@ def service_params
                         :intentions_required, :buyers_manage_apps, :referrer_filters_required,
                         :buyer_can_select_plan, :buyer_plan_change_permission, :buyers_manage_keys,
                         :buyer_key_regenerate_enabled, :mandatory_app_key, :custom_keys_enabled, :state_event,
-                        :txt_support, :terms,
+                        :txt_support, :terms, :annotations,
```

Comment:
Perhaps when having tests it will show by itself, but I think it needs to be something like `annotations: {}` or idk off the top of my head.

Reply:
I see in `providers_controller` it is `{annotations: %i[name value]}]`.

Reply:
also we have `null: false` in table creation